adapter/pkg/config/parser.go

```diff
@@ -58,6 +62,23 @@ func GetMgwHome() string {
 	return mgwHome
 }
 
+// getLogConfigPath reads the LOG_CONFIG_PATH environmental variable and returns the value.
+// If the env variable is not available, returned value would be the combination of MGW_HOME
+// env variable value + relative log config path
+func getLogConfigPath() string {
+	logConfigPath = os.Getenv(logConfigPathEnvVariable)
+	if len(strings.TrimSpace(logConfigPath)) == 0 {
+		//for backward compatibility
+		logConfigPath = GetMgwHome() + relativeLogConfigPath
+	}
+	_, err := os.Stat(logConfigPath)
+	if err != nil {
+		logger.Fatal("Log configuration file not found.", err)
+		panic(err)
```

Comment:
No need to panic if the the log_config.toml is not available
And log level should be set to Warn